client/webpack/base.config.js

```diff
@@ -50,7 +50,13 @@ module.exports = (options = {}) => {
     use: [
       { loader: "style-loader" },
       { loader: "css-loader", options: "importLoaders=2" },
-      { loader: "postcss-loader", options: "syntax=postcss-scss" },
+      {
+        loader: "postcss-loader",
+        options: {
+          syntax: "postcss-scss",
+          plugins: () => [require("autoprefixer")]
```

Comment:
You're requiring a module that isn't explicitly included in our package.json file. It works, because autoprefixer happens to be a dependency of css-loader. But, while it works, it's not correct. Add auto-prefixer to our package.json file to address this.